### DISCUSSION TITLE
From what language did English borrow the word "ketchup"?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#307] [Galaxy Trivia] [answered]
**Hint:** 🇲🇾

**Comment by anggerpr** [comment]
Ketchup may have entered the English language from the Malay word kicap (pronounced [kitʃap], sometimes spelled kecap or ketjap). Originally meaning "soy sauce", the word itself derives from the hokkien (a Chinese dialect) word “ke-tsiap”.

**Comment by thuongtruong1009** [comment]
The English word "ketchup" is believed to have been borrowed from the Malay word "kechap" (also spelled "kicap" or "ketjap"), which means "sauce" or "condiment". The Malay word itself is thought to have been derived from the Chinese word "ketsiap", which refers to a sauce made from fermented fish or soybeans.

**Comment by ItsNotAlexy** [comment, marked as answer]
The answer is "Malay", From the word "kejap" or "kicap". Fun fact, Did you also know that Indonesia has the same word in malay? "Kecap" is not ketchup but a very sweet sauce just like soy sauce!

**Reply by Mithun1508** [reply]
Ketchup” Is Borrowed From Hokkien And Malay Words.

**Reply by ParanoidUser** [reply]
Well done! 👏👏 Congratulations on your well deserved [Galaxy Brain](https://github.com/ItsNotAlexy?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Bronze for 8 answers), and don't hesitate to post your own questions to the community. 😊🎉
